src/main/java/nl/stil4m/transmission/api/domain/AddedTorrentInfo.java

```diff
@@ -6,14 +6,29 @@
 public class AddedTorrentInfo {
 
 
-    private final TorrentInfo torrentAdded;
+    private final TorrentInfo torrentInfo;
+
+    private boolean isDuplicate = false;
```

Comment:
There is only 1 constructor. Can you set it the constructor and make the field final?